src/main.ts

```diff
@@ -15,8 +17,26 @@ export async function run() {
   let formattedCommand = ''
 
   try {
+    const osName = process.platform
+    let extractZippedFilePath: string = inputs.SYNOPSYS_BRIDGE_PATH || getBridgeDefaultPath()
+
     // Automatically configure bridge if Bridge download url is provided
-    if (inputs.BRIDGE_DOWNLOAD_URL) {
+    if (inputs.CONFIGURE_FROM_REPO && inputs.CONFIGURE_FROM_REPO.toLowerCase() === 'true') {
+      info('Configuring Bridge from synopsys-action repository')
+      let configFilePath = path.join(getWorkSpaceDirectory(), 'bridge')
+      let availableFileName = ''
+      if (osName === 'darwin') {
+        availableFileName = getRequiredFileNameWithPattern(configFilePath, 'mac')
```

Comment:
can we create a varriable paltform and pass it in getRequiredFileNameWithPattern 
example: let paltform='';
if (osName=='darwin'){
platform = mac;
}
else if (osName= 'windows")
{
platform= win
}

availableFileName = getRequiredFileNameWithPattern(configFilePath, platform) will be called once. cleaner code